roles/kubernetes/preinstall/defaults/main.yml

```diff
@@ -98,6 +98,12 @@ ntp_servers:
 ntp_restrict:
   - "127.0.0.1"
   - "::1"
+# Specify whether to filter interfaces
```

Comment:
HI @Pavan-Gunda 

Because there is only one variable that needs to be config, so 

```
# Some comments
# Uncomment `ntp_interfaces` if enable xxxx
# ntp_interfaces: 
#   - ignore wildcard
#   - isten xxx
```

Which Can be easier for the code.

Reply:
Thank you :+1: 

I made the change now and pushed the code :)